automation/tests/test_backport_install.py

```diff
@@ -0,0 +1,88 @@
+        cls.CLUSTER_OPERATOR_DEPLOYMENT_FILE = cls.RESOURCES_PATH + "test.060-Deployment-strimzi-cluster-operator.yaml"
+        cls.DRAIN_CLEANER_OPENSHIFT_DEPLOYMENT_FILE = cls.RESOURCES_PATH + "test.060-Deployment-strimzi-drain-cleaner-openshift.yaml"
+
+    def _assert_yaml_fields(self, yaml_data, expected_fields):
+        image_field = yaml_data["spec"]["template"]["spec"]["containers"][0]["image"]
+        label_field = yaml_data["spec"]["template"]["metadata"]["labels"]
+        self.assertEqual(image_field, expected_fields)
+        self.assertEqual(label_field["rht.comp_ver"], self.RHT_COMP_VERSION)
+        self.assertEqual(label_field["rht.prod_ver"], self.YEAR_AND_QUARTER_VERSION)
+
+    def test_yaml_parse(self):
+        yaml_data = _load_yaml_data(self.USER_OPERATOR_DEPLOYMENT_FILE)
+        expected_metadata = {'app': 'strimzi'}
+        self.assertEqual(yaml_data["metadata"]["labels"], expected_metadata)
+
+    def test_update_cluster_operator_deployment(self):
+        update_cluster_operator_deployment(self.CLUSTER_OPERATOR_DEPLOYMENT_FILE, self.STRIMZI_VERSION)
+        yaml_data = _load_yaml_data(self.CLUSTER_OPERATOR_DEPLOYMENT_FILE)
+        env_field = yaml_data["spec"]["template"]["spec"]["containers"][0]["env"][7]["value"]
+        expected_image = f"{self.COMMON_REGISTRY_URL}strimzi-rhel8-operator:{self.STRIMZI_VERSION}"
+        self.assertIn(self.EXPECTED_NEWEST_KAFKA_VERSION, env_field)
+        self.assertIn(self.EXPECTED_UPGRADE_KAFKA_VERSION, env_field)
+        self._assert_yaml_fields(yaml_data, expected_image)
+
+    def test_update_user_deployment(self):
+        update_user_deployment(self.USER_OPERATOR_DEPLOYMENT_FILE, self.STRIMZI_VERSION)
+        yaml_data = _load_yaml_data(self.USER_OPERATOR_DEPLOYMENT_FILE)
+        expected_image = f"{self.COMMON_REGISTRY_URL}strimzi-rhel8-operator:{self.STRIMZI_VERSION}"
+        self._assert_yaml_fields(yaml_data, expected_image)
+
+    def test_update_topic_deployment(self):
+        update_topic_deployment(self.TOPIC_OPERATOR_DEPLOYMENT_FILE, self.STRIMZI_VERSION)
+        yaml_data = _load_yaml_data(self.TOPIC_OPERATOR_DEPLOYMENT_FILE)
+        expected_image = f"{self.COMMON_REGISTRY_URL}strimzi-rhel8-operator:{self.STRIMZI_VERSION}"
+        self._assert_yaml_fields(yaml_data, expected_image)
+
+    def test_update_drain_cleaner_deployment(self):
+        update_drain_cleaner_deployment(self.DRAIN_CLEANER_OPENSHIFT_DEPLOYMENT_FILE, self.STRIMZI_VERSION)
+        yaml_data = _load_yaml_data(self.DRAIN_CLEANER_OPENSHIFT_DEPLOYMENT_FILE)
+        expected_image = f"{self.COMMON_REGISTRY_URL}drain-cleaner-rhel8:{self.STRIMZI_VERSION}"
+        self._assert_yaml_fields(yaml_data, expected_image)
+
+    def test_get_last_year_and_quarter(self):
+        expected_year = 2023
+        expected_quarter = 4
+        year, quarter = get_last_year_and_quarter()
+        self.assertEqual(year, expected_year)
+        self.assertEqual(quarter, expected_quarter)
```

Comment:
Be careful with this test, it will automatically break every few months if the test isn't updated